integrations/hyperquery.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: Hyperquery
+parent: Integrations
+---
+
+# Using GlareDB in Hyperquery
+
+[Hyperquery] is a data notebook built for speed, visibility, and collaboration.
+Hyperquery provides the ability to write and execute SQL queries or Python cells
+directly within a rich graphical interface.
+
+## Running GlareDB in Hyperquery
+
+To get started, select an existing notebook, or create a new one.
+
+To install GlareDB in the notebook type `/`, then select **Python block**. This
+will place a new block for executing python in the notebook, which will be used
+to to install the [glaredb python library].
+
+![python block]
+
+Execute the following command in the code block:
+
+```shell
+pip install glaredb
+```
+
+Next, in a new **Python block**, import and use glaredb:
+
+```python
+import glaredb
+
+con = glaredb.connect()
+con.sql("SELECT 'hello from hyperquery';").to_pandas()
+```
+
+{: .important }
+
+> [Pandas] is in scope. GlareDB provides interop with both [Pandas] and [Polars]
+> dataframe libraries. For more information, refer to [GlareDB Python documentation].
+
+Executing `SELECT 'hello from hyperquery';` will produce output like:
+
+![success]
+
+## Connecting to GlareDB Cloud from Hyperquery
+
+Sign in or sign up with [GlareDB Cloud]. Upon signing up, a free deployment is
+instantly created and ready to be used. Obtain the Python connection details for
+a deployment by clicking the **Connect** button and selecting the **Python** tab.
```

Comment:
I gave feedback for how we can improve these cloud primers above for Fabric; I think we should copy-paste the same fine-tuned primer + connection instructions here